### PR TITLE
add new recipe lcr_genie v 1.0.1

### PR DESCRIPTION
**Pull request by kenza12**
**LCRGenie** : Ligase Chain Reaction

development repository: https://github.com/neilswainston/LCRGenie/

Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kenza12** [comment]
@conda-forge-admin, please ping team

**Comment by breakthewall** [comment]
Hi! I'm one of the maintainers of this recipe. Thanks!

**Review by beckermr — CHANGES_REQUESTED** [review]
We cannot accept pre-built binaries. The missing DSO whitelist needed means the package likely has them.

**Comment by kenza12** [comment]
@conda-forge-admin, please ping team

**Comment by kenza12** [comment]
@conda-forge-admin, please ping team

File: recipes/lcr_genie/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/neilswainston/LCRGenie/archive/refs/tags/{{ version }}.tar.gz
+  sha256: f84a26c926cb02f343f79b2facf25bec3ac03a0e337f9bd83a47165c8bccfa68
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install -vv .
+  skip: true  # [win or py != 37]
```
Comment (dopplershift) [inline recipes/lcr_genie/meta.yaml:15]: This is pure Python and all dependencies seem to be available? Why is this skipping Windows and why isn't it just made `noarch: python`?

Comment (kenza12) [reply recipes/lcr_genie/meta.yaml]: Hi @dopplershift ,
You're right, I don't know why it didn't work before on windows.
Thank you !!

**Pull request merged**